frontend/src/components/Dashboard/Header/UserControls.js

```diff
@@ -1,56 +1,29 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-import { withRouter } from 'react-router-dom';
 import auth from 'services/gqlAuth';
-import classnames from 'classnames';
 
-import cogSvg from 'images/cog.svg';
 import 'components/Dashboard/Header/UserControls.css';
 
-class UserControlsBase extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      showControls: false,
-    };
-  }
-
-  toggleControls = () => {
-    this.setState({ showControls: !this.state.showControls });
-  };
-
+export default class UserControls extends React.Component {
   render() {
-    return auth.isAuthenticated() ? (
-      <div className={classnames(this.props.cqParams, 'UserControls')}>
-        <img
-          src={cogSvg}
-          alt="Link User Settings"
-          className="Header__settings-icon"
-          onClick={this.toggleControls}
-        />
-        {(this.props.cqParams.fullsize || this.state.showControls) &&
-        this.props.currentUser ? (
-          <Link className="UserControls__text" to="#">
+    return auth.isAuthenticated ? (
+      <div className="UserControls">
+        {this.props.currentUser && (
+          <Link className="UserControls__link" to="#">
```

Comment:
Maybe we should change this to not be a link until we have a user details page or something of the sort. I don't think we have anything planned that clicking a username makes sense for yet.

Reply:
Killing it based on our discussion yesterday, since we don't have any user profile page.